browser/resources/settings/brave_overrides/privacy_page.ts

```diff
@@ -42,20 +43,27 @@ function InsertGoogleSignInSubpage (
       console.error(
         '[Brave Settings Overrides] Couldn\'t find Google signin subpage')
     } else {
-      googleSignInSubpage.setAttribute(
-          'page-title',
+      const pageTitle =
+        googleSignInTemplate.content.getElementById('page-title')
+      if (!pageTitle) {
+        console.error(
+          '[Brave Settings Overrides] Couldn\'t find Google sigin page-title')
```

Comment:
```suggestion
          '[Brave Settings Overrides] Couldn\'t find Google signin page-title')
```

nit.

Reply:
If this is all we find in this review, I think I'll let it be so that we can get the builds out quickly. Will make a note to fix it as a follow-up though.